cli.ts

```diff
@@ -192,53 +195,68 @@ async function download(root?: string, dest = "./bin"): Promise<string> {
   return executable;
 }
 
-// FIXME: Add support for .jsonc files
-async function readDenoJson(denoJsonPath: string) {
+/**
+ * Reads and parses deno.json or deno.jsonc whichever found first. Returns both the parsed content
+ * and the path to actual file discovered.
+ */
+async function readDenoJson(
+  resolvedDirectory: string,
+  extension: "json" | "jsonc" = "json",
+) {
   try {
-    const denoJson = JSON.parse(await Deno.readTextFile(denoJsonPath));
-    return denoJson;
+    const denoJsonPath = join(resolvedDirectory, `deno.${extension}`);
+    const contents = await Deno.readTextFile(denoJsonPath);
+    return {
+      denoJson: extension === "json"
+        ? JSON.parse(contents)
+        : JSONC.parse(contents),
+      denoJsonPath,
+    };
   } catch (err) {
-    if (err instanceof Deno.errors.NotFound) {
-      throw new Error(
-        "Could not find deno.json in the specified working directory.",
-      );
+    if (!(err instanceof Deno.errors.NotFound)) {
+      throw err;
     }
 
-    throw err;
+    return null;
   }
 }
 
 /**
  * Add Tailwind CLI commands to Deno tasks.
- * @todo Add support for .jsonc files
- * @param cwd - Current working directory for Deno project.
+ * @param cwd - Current working directory for Deno project. Defaults to `"./"` if undefined.
  */
 export async function addTask(cwd?: string) {
-  const denoJsonPath = join(cwd ?? "./", "deno.json");
-  // Add task to deno.json
+  // Add task to deno.json[c]
   try {
-    const denoJson = await readDenoJson(denoJsonPath);
+    const denoConfig = (await readDenoJson(cwd ?? "./", "json")) ??
+      (await readDenoJson(cwd ?? "./", "jsonc"));
+
+    if (!denoConfig) {
+      throw new Error(
+        "Could not find deno.json or deno.jsonc in the current working directory.",
+      );
+    }
+
+    const { denoJson, denoJsonPath } = denoConfig;
 
     denoJson.tasks.tailwind = "./bin/tailwindcss";
 
     if (!denoJson.tasks["tailwind:build"]) {
       denoJson.tasks["tailwind:build"] =
-        "./bin/tailwindcss -i ./src/style.css -o ./static/style.css --config ./tailwind.config.ts --minify";
+        `./bin/tailwindcss -i ${DEFAULT_STYLE_SRC} -o ${DEFAULT_STYLE_DEST} --config ./tailwind.config.ts --minify`;
     }
 
     if (!denoJson.tasks["tailwind:watch"]) {
       denoJson.tasks["tailwind:watch"] =
-        "./bin/tailwindcss -i ./src/style.css -o ./static/style.css --config ./tailwind.config.ts --watch";
+        `./bin/tailwindcss -i ${DEFAULT_STYLE_SRC} -o ${DEFAULT_STYLE_DEST} --config ./tailwind.config.ts --watch`;
     }
 
-    await Deno.writeTextFile(denoJsonPath, JSON.stringify(denoJson, null, 2));
+    // FIXME: Using JSON.stringify will remove comments from deno.jsonc. The current workaround is to use JSON.stringify to always create a deno.json file
```

Comment:
Writing _back_ to .jsonc files is currently an issue. `JSONC.stringify` does not exist, so the `JSON.stringify` function would end up stripping out the comments. My current solution is to just write to a deno.json anyway, and expect devs to copy that into their deno.jsonc file.